pkg/kubelet/kubelet.go

```diff
@@ -1709,7 +1779,11 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Wait for volumes to attach/mount
-	if err := kl.volumeManager.WaitForAttachAndMount(pod); err != nil {
+	defaultedPod, _, err := kl.defaultPodLimitsForDownwardApi(pod, nil)
+	if err != nil {
+		return err
+	}
+	if err := kl.volumeManager.WaitForAttachAndMount(defaultedPod); err != nil {
```

Comment:
i *thought* defaulted pod was a copy.  is there an issue if its a copy here?

actually, i am not following why your PR makes this change at all.